src/Asset/EntrypointLookup.php

```diff
@@ -100,6 +122,33 @@ private function getEntriesData(): array
             }
         }
 
+        if ($this->cache) {
+            $cached->set($this->entriesData);
+            $this->cache->save($cached);
+        }
         return $this->entriesData;
     }
+
+    private function cacheConfigJsonFile(): ConfigCacheInterface
+    {
+        return $this->getConfigCacheFactory()->cache($this->getConfigCacheFileName(), function (ConfigCacheInterface $cache) {
+            $content = sprintf(<<<EOF
+<?php
+return %s;
+EOF
+                , var_export(json_decode(file_get_contents($this->entrypointJsonPath), true), true));
+
+            $cache->write($content, [new FileResource($this->entrypointJsonPath)]);
+        });
+    }
+
+    public function warmUp($cacheDir)
```

Comment:
The warmup should now use the cache service to read and set the cache - no more ConfigCacheFactory stuff